### PR TITLE
ADEN-2293 Wrap JS commands into one function

### PR DESCRIPTION
**Pull request by jacekjursza**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/5cc828c0e9319b3548ab5743175d2e46f8934870/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L705) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/5cc828c0e9319b3548ab5743175d2e46f8934870/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java#L613) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by nandy-andy** [comment]
LGTM but before merging let's try fix sonar issues or discuss them on daily ;) Also the change is in `AdsBaseObject` so the only thing left from our convention is to make sure if the Jenkins job is named correctly.

**Comment by drets** [comment]
Explanation of Sonar issues:
http://stackoverflow.com/questions/29563884/exception-handlers-should-preserve-the-original-exception/29564067#29564067

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 1]
```diff
@@ -698,12 +698,12 @@ private void tryTriggerFloatingMedrec(final WebElement slot) {
         @Override
         public Object apply(WebDriver webDriver) {
           jsActions.execute(
-              "window.scroll(0, 5000); setTimeout(function () {window.scroll(0, 5001) }, 500)");
+              " (function(){ window.scroll(0, 5000); setTimeout(function () {window.scroll(0, 5001) }, 100); })(); ");
           return slot.getAttribute("style").contains("visibility: visible;");
         }
       });
     } catch (org.openqa.selenium.TimeoutException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: MAJOR Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 2]
```diff
@@ -611,7 +611,7 @@ public Boolean apply(WebDriver driver) {
             return (boolean) ((JavascriptExecutor) driver)
                 .executeScript("return !!(" + script + ");");
           } catch (WebDriverException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: MAJOR Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 17 issues:
* MAJOR 11 major
* MINOR 6 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "</td><td>" 4 times.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L70) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "<tr class=\"" 4 times.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L70) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Introduce a new variable instead of reusing the parameter "imageAsBase64".](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L142) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1226)
* MAJOR [Remove this unused "date" local variable.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L310) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L135) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Remove this unused "dateFormat" local variable.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L309) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L344) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)
* MAJOR [Move this closing curly brace to the next line.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L303) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)
* MINOR [Define a constant instead of duplicating this literal "success" 3 times.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L68) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L303) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)
* MINOR [Define a constant instead of duplicating this literal "\"><td>" 4 times.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L70) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L144) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L343) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "error" 3 times.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L68) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "</td><td> <br/> &nbsp;</td></tr>" 8 times.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L108) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L269) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Replace this usage of System.out or System.err by a logger.](https://github.com/Wikia/selenium-tests/blob/2603e412c7a1e626002ec0954d168d04bfec9b63/src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java#L245) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)

**Comment by nandy-andy** [comment]
LGTM :+1:

**Pull request merged**